docs/marketplace/fastlane/testing-distribution/index.md

```diff
@@ -31,19 +49,30 @@ fastlane add_plugin appcircle_testing_distribution
 
 ```yml
   appcircle_testing_distribution(
-    accessToken: "$(AC_ACCESS_TOKEN)", # Your Appcircle Access Token
-    profileID: "$(AC_PROFILE_ID)", # ID of your Appcircle Distribution Profile
-    appPath: "$(AC_APP_PATH)", # Path to your iOS .ipa or .xcarchive, or Android APK or App Bundle
-    message: "$(AC_MESSAGE)", # Your Message
+    personalAPIToken: "$(AC_PERSONAL_API_TOKEN)",
+    profileName: "$(AC_PROFILE_NAME)",
+    createProfileIfNotExists: Boolean,
+    appPath: "$(AC_APP_PATH)",
+    message: "$(AC_MESSAGE)",
   )
 ```
 
+- `personalAPIToken`: The Appcircle Personal API token is used to authenticate and secure access to Appcircle services. Add this token to your credentials to enable its use in your pipeline and ensure authorized actions within the platform.
+- `profileName`: Specifies the profile that will be used for uploading the app.
+- `createProfileIfNotExists`: Ensures that a user profile is automatically created if it does not already exist; if the profile name already exists, the app will be uploaded to that existing profile instead.
+- `appPath`: Indicates the file path to the application that will be uploaded to Appcircle Testing Distribution Profile.
+- `message`: Your message to testers, ensuring they receive important updates and information regarding the application.
+
 :::caution Build Steps Order
 You should add this task extension after completing your build steps.
```

Comment:
Same updates with EAS: "Ensure that this action is added after build steps have been completed."